sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
-const models = require('./modelSetup');
+const models = require("./modelSetup");
 
 //---------------FindAll: Get all data from the table --------------//
 //--------NOTE: These functions return Promises. Use ".then()" after calling them.
 
 getAllUsers = () => {
-    return models.User.findAll().then((users) => {
-        return users;
-    }).catch(err => console.err(err));
+  return models.User.findAll()
```

Comment:
It seems many changes marked in this file were from format changes between users.